docs/pages/guides/environment-variables.mdx

```diff
@@ -59,16 +61,31 @@ If you create an environment-specific file, like **.env.test**, you can load it
 
 <Terminal cmd={['$ NODE_ENV=production npx expo start']} />
 
+### Disabling environment variables
+
+Environment variables in Expo CLI have two parts, both can be disabled:
```

Comment:
```suggestion
Environment variables in Expo CLI have two parts and both can be disabled:
```

Using a `,` is not a separator. Let's break this down into two sentences or use conjunction like the above suggestion.